class.jetpack-gutenberg.php

```diff
@@ -32,6 +32,19 @@ function jetpack_register_block( $slug, $args = array() ) {
 		return false;
 	}
 
+	// If the block is dynamic, and a Jetpack block, wrap the render_callback to check availability.
+	if ( 0 === strpos( $slug, 'jetpack/' ) && isset( $args['render_callback'] ) ) {
```

Comment:
This is a bit hard to grok, maybe it should be in a different function?

Reply:
Definitely, I'll extract it and try to make it a bit more readable.

Reply:
Looks way better now :P)